episodes/03-create.md

```diff
@@ -94,9 +219,15 @@ $ git checkout -b main
 ```output
 Switched to a new branch 'main'
 ```
+-->
+
+## Check the status
 
+To interact with Git, we can also use the [Rstudio Terminal](https://docs.posit.co/ide/user/ide/guide/tools/terminal.html). 
+
+In the Rstudio Terminal, 
 We can check that everything is set up correctly
```

Comment:
```suggestion
In the RStudio Terminal, we can check that everything is set up correctly
```

Reply:
for context, in some lines I edited over the [Semantic Line Breaks](https://sembr.org/) that the original repo used. So is highly likely to find them elsewhere in the document, except in the writings I made. The workbench includes them in their style guide, but do not include it in my writing yet. ref: https://carpentries.github.io/sandpaper-docs/style.html#line-length

Reply:
for context, in some lines I edited over the [Semantic Line Breaks](https://sembr.org/) that the original repo used. So is highly likely to find them elsewhere in the document, except in the writings I made. The workbench includes it in their style guide, but I have not included it in my writing yet. ref: https://carpentries.github.io/sandpaper-docs/style.html#line-length